### PR TITLE
updates helm charts

### PR DESCRIPTION
**Pull request by JBAhire**
## Description
This PR, 
- Updates helm charts as per the latest releases of all services. 

### Testing
I have deployed Hypertrace using this versions locally and everything is working fine.
Screenshot 2020-11-11 at 4 20 27 PM

### Checklist:
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: kubernetes/platform-services/Chart.yaml [hunk 1]
```diff
@@ -25,41 +25,37 @@ appVersion: 0.1.0
 dependencies:
   - name: hypertrace-oc-collector
     repository: "https://storage.googleapis.com/hypertrace-helm-charts"
-    version: 0.1.5
+    version: 0.1.6
   - name: span-normalizer
     repository: "https://storage.googleapis.com/hypertrace-helm-charts"
-    version: 0.1.23
+    version: 0.3.0
```
Comment (kotharironak) [inline kubernetes/platform-services/Chart.yaml:31]: All ingestion with the same version. good one.

**Review by kotharironak — APPROVED** [review]

File: kubernetes/platform-services/Chart.yaml (hunk 1 above)
Comment (JBAhire) [reply kubernetes/platform-services/Chart.yaml:31]: yeah! after `hypertrace-service` it will become more easier!

**Comment by jcchavezs** [comment]
From time to time I see this @JBAhire bot updating helm charts, changing versions here and there. I wonder if we could:

1. Abstract versions in a single file
2. Create a script that fetches that info from container registry and update the versions accordingly.
3. Automate this as a nightly workflow so we get notified when there is a new version so we run this either locally or trigger a PR (it could be a nice github action, https://github.com/marketplace/actions/create-pull-request)

**Comment by JBAhire** [comment]
Sure, @jcchavezs ! I will look into this! :)

**Pull request merged**